apps/src/code-studio/components/progress/ScriptOverviewTopRow.jsx

```diff
@@ -105,6 +112,12 @@ export default class ScriptOverviewTopRow extends React.Component {
               assignmentName={scriptTitle}
             />
           )}
+        {experiments.isEnabled(experiments.ASSIGNMENT_UPDATES) && (
+          <SectionAssigner
+            sections={sections}
+            initialSelectedSectionId={selectedSectionId}
```

Comment:
please hide the old assign button when the experiment is enabled

Reply:
Gah! Sorry I should've caught that after the last reminder. 
<img width="1026" alt="Screen Shot 2019-10-18 at 3 31 06 PM" src="https://user-images.githubusercontent.com/12300669/67132141-7fb19180-f1bc-11e9-820b-7dd3af9aef60.png">